doc/content/verification_and_validation/val-2f.md

```diff
@@ -0,0 +1,138 @@
+
+!table id=val-2f_traps_values caption=Values of traps parameters for 0.1 dpa.
+| Parameter | Description                            | Value                                                       | Units                 | Reference                 |
+| --------- | ------------------------------------   | ----------------------------------------------------------- | --------------------- | --------------------- |
+| $\nu_{\mathrm{dt},0}$ | Traps de-trapping prefactor | $10^{13}$                                           | atoms/s               | [!cite](dark2024modelling) |
+| $E_{\mathrm{dt},1}$ | Trap 1 de-trapping energy     | 1.15                                                        | eV                    | [!cite](dark2024modelling) |
+| $n_1$ | Trap 1 density                             | 4.8$\times 10^{25}$                                         | atoms/m$^3$           | [!cite](dark2024modelling) |
+| $E_{\mathrm{dt},2}$ | Trap 2 de-trapping energy     | 1.35                                                        | eV                    | [!cite](dark2024modelling) |
+| $n_2$ | Trap 2 density                             | 3.8$\times 10^{25}$                                         | atoms/m$^3$           | [!cite](dark2024modelling) |
+
+## Results
+
+[val-2f_comparison] shows the comparison of the TMAP8 calculation and the experimental data during desorption. The experimental data are provided by T. Schwarz-Selinger and are available [here](https://zenodo.org/records/11085134). The distinctive shape of the TMAP8 curve arises from the detrapping energies of trap 1 and trap 2. By including additional traps in the simulations, the TMAP8 results are expected to better align with the experimental data.
+
+!media comparison_val-2f.py
+       image_name=val-2f_comparison.png
+       style=width:50%;margin-bottom:2%;margin-left:auto;margin-right:auto
+       id=val-2f_comparison
+       caption=Comparison of TMAP8 calculations with experimental data on deuterium flux (atoms/m$^2$/s) for a damage of 0.1 dpa.
+
+[val-2f_deuterium_desorption] displays the quantities of mobile, trapped, and escaping deuterium atoms during the desorption process. During desorption, the temperature increases from 300K to 1000K. The amount of deuterium trapped will decrease as the temperature rises and the various trapping energies are reached. Trap 1 will release deuterium before trap 2, as its trapping energy is smaller.
+
+During desorption, no further implantation occurs, resulting in a decrease in the number of mobile and trapped deuterium atoms and an increase in the number of escaping deuterium atoms. Mass conservation is well maintained during desorption, with only a 0.02% error between the initial number of mobile and trapped deuterium atoms and the total number of deuterium atoms (mobile, trapped and escaping).
+
+!media comparison_val-2f.py
+       image_name=val-2f_deuterium_desorption.png
```

Comment:
It seems like you are missing the mobile deuterium, and the legend needs to be fixed for it too.